### ISSUE TITLE
BibTex editor cannot open

### ISSUE THREAD

**Issue by lijunzh** [JabRef/jabref#3371]
JabRef version  on OSX 10.12.6 

After upgrading to JabRef 4.0, I have problem open the editor of BibTeX. I can see all entries and right click on them. But, when I trying to update the old entry or creating a new entry, the editor cannot be opened and the entire JabRef froze for a long time until I force it to quit. I updated my java to the latest and the problem is still there.

Let me know if you need any kind of log file or screen shots. I will need some instructions for how to find those log files though.

**Comment by tobiasdiez** [comment]
A few users already reported problems with JabRef 4.0 on MacOS. Most of them could be solved by adding something to the `host` file, see https://help.jabref.org/en/FAQosx#q-jabref-slowhangs-on-macos-sierra.

Thus this issue is probably a duplicate of https://github.com/JabRef/jabref/issues/3236 or https://github.com/JabRef/jabref/issues/3298.

**Comment by lijunzh** [comment]
@tobiasdiez Thanks for the info. After following

> Most of them could be solved by adding something to the host file, see https://help.jabref.org/en/FAQosx#q-jabref-slowhangs-on-macos-sierra.

the problem improves. Now, Jabref opens the editor after about 2 to 3 minutes. It works fine when I am inside the editor; however, it froze again when I tried to switch entry or re-enter editor. 

Note: I did reboot my computer to make sure all changes in /etc/host take effects.

I switched back to [JabRef 3.8.2](https://github.com/JabRef/jabref/releases/tag/v3.8.2), the problem disappears. I think I will stick to 3.8.2 since waiting for 2-3 minutes for just changing a character does not make sense. I can, of course, edit the bibtex file in other text editors. But this makes JabRef only half useful to me.
